src/Alerts/MessageContext.tsx

```diff
@@ -0,0 +1,104 @@
+import { createContext, Reducer } from 'react';
+/**
+ * Global message provider
+ */
+
+export const MessageContext = createContext(null);
```

Comment:
Also, in `mark-one`, we have a convention where there isn't an extra empty line between comments and code. Do you think we could update the formatting so that there aren't these extra line breaks?

Reply:
Sorry, looks like I made a mistake. I believe the typing should include `DispatchMessage` like so as it is in Course Planner:
```
/**
 * A function that passes down a message to the queue
 */
export type DispatchMessage = Dispatch<MessageReducerAction>;

/**
 * Global message provider
 */
export const MessageContext = createContext<DispatchMessage>(null);
```